src/test/test_FKi.py

```diff
@@ -0,0 +1,51 @@
+#! /usr/bin/env python
+import rospy
+tests = 'test_roslaunch'
+import unittest
+from std_srvs.srv import Trigger, TriggerRequest, TriggerResponse
+import numpy as np
+import time
+
+b = []
+c = []
+
+import tf
+import geometry_msgs.msg
+t = tf.Transformer(True, rospy.Duration(10.0))
+t.getFrameStrings()
+[]
+m = geometry_msgs.msg.TransformStamped()
+m.header.frame_id = 'world'
+m.child_frame_id = 'camera_link'
+xr = m.transform.translation.x
```

Comment:
i did not catch what you are doing here. Seems like some initialization and xr is just zero.